data/ssystem_minor.ini

```diff
@@ -1822,6 +2018,7 @@ dust_lengthfactor              = 0.4
 dust_widthfactor               = 1.5
 model                          = 1682q1halley_MLfix.obj
 name                           = 1P/Halley (2061)
+iau_designation                = P/1682 Q1
```

Comment:
I am unsure about using the 1682 label to the future apparition. Do recurring comets keep the label of earlier apparition? I know, WP is not the most reliable source, but they provide a list of new designations for 1P/Halley in all known apparitions.

Reply:
We really need development the rule for adding the extra parameters for periodic comets, such, `*_designation`, ` *_code` and `discover*` because we should follow historical data in the one hand and resolve messy issues in the other hand. Plus maybe we should all some extra parameters for comets, like `last_seen` for unperiodical comets.

I hope I'll find the monography _Физические характеристики комет_ and all appticaltions for it and all volumes of _Cometography_ to adding missing data and fix mistakes. Anyone can help me find these books?

Reply:
last_seen can only be useful for periodical comets which were lost and now should carry D/  labels ;-) If really the first "modern" discovery code (YYYY H1) is used for periodical comets, we need to specify which apparition we are describing. For now, we do this as part of the name (e.g. "1P/Halley (2061)". But who then specified the neo-historical Halley date codes?  Of course, the apparition is coded in the date of perihel, but humans rarely decode JD directly.